starter-code/app.js

```diff
@@ -15,23 +30,54 @@ const siteController = require("./routes/siteController");
 mongoose.connect("mongodb://localhost/ibi-ironhack");
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'ejs');
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "ejs");
 
 // uncomment after placing your favicon in /public
 //app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
-app.use(logger('combined'));
+app.use(logger("combined"));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, "public")));
 
 // Routes
 app.use("/", siteController);
```

Comment:
If you `use` your routes/controller before configuring and `use`ing passport, then you won't be able to authenticate anything before navigating to a page.

Reply:
Your controllers always come at the end, before the error handlers